### DISCUSSION TITLE
Is the backend secure, and how to secure it?

### DISCUSSION THREAD

**Discussion by NurdinDev** [amplication/amplication#1333] [Q&A]
Hello Amplication :) 
I was trying Amplication and I'm wondering about the situation of the security side and how Amplication secure his APIs without JWT

**Comment by yuval-hazaz** [comment]
Hi @NurdinDev 
I assume you are talking about your generated application (and not about Amplication server).
The generated app is using BasicHttp authentication with a username and password that are sent in the header of every request. 
(https://en.wikipedia.org/wiki/Basic_access_authentication)

We are planning to add support for more authentication providers like OAuth2
You can comment and track this issue https://github.com/amplication/amplication/issues/1084

If you have any specific use case or request in mind please share.

**Reply by NurdinDev** [reply]
Thank you @yuval-hazaz 👍
